.travis.yml

```diff
@@ -4,7 +4,7 @@ deploy:
   email: opensource@realytics.io
   skip_cleanup: true
   api_key:
-    secure: RdoLwtLDz3PI8fhTg0wXeRuL9JAffNJSTJvNExk9Q+fcccBl8yaNU4AId5OcuDwO3pnb59nKv8fIidW9VzemZ8/eCqCBbCx4qzGFxN2uCB+aBwQ7Svu6GAJV0JIEgIXG+dkByRawd9hCIyvi6IvhIA/XbVbboV5ngjwSW317SA0lbznGHQmIT4o93+YpUuT+pkGC04eJGIH6cgAe61QGwN6WyGx8hfJ+K+OCksZ5A4RIeSkRHY4zRgmlGGtSwOutUPh3OMIdG40E0UXLn3MRdANzekMFSowMZ8aFliqhDsgk/zk8IC/X6OlPFSTAmO1IkKqZiJsr1rxmeJvC9GDxvRZ3Vyv3GncoteEQVjKFsYRlTvmEouECSxafl2aOqa1i2TqU8if5GjgB6mm+pTjDTzF4KB3QkEtRKgTuB47+EUc5zXIBrgIAb3sWu+QgaSFOreG49cGccCsU0viLe2WHbjzuNUP3IMO0x9tL/rm0RQmP3FjQSWlDg6VXYjJs5UcpiQLabgL3u4tso9aYprqTKihqfQkyfi0R5qH872ncGlASkzwbcZxiUeWau3yx46iZdckPNTAKw0afeodmr7aL6B+xEWHfZv9OBn0UE5I23jvIlI+wzyWHWUgeVikBx1ZZV4OqAHOTdEMHQivbir4eYGGAYtY6yxzLpyfD4g0xIzM=
+    secure: J4q7IGixHsEp5of9FG/Zefit8ba6bV7T1I0yV8TBNQHfwBaTC344PBhx3jRg8Nhv04vzkzY+s2cJYIuyqlfEeZrsBcFTvPE9LowR/4cMCoFjteb/gDGtW2AFw2M5pfJ6L4PHXtigO5emWs7xeT3R8KxcQn3e2nzJcGXShc0N9vCCDe601MKJFco5svoHqx5cAmHoknIQDPatddob9VdOUvaRGZWWh7Pv8pdDHGMcUk18TL67p55vMDXL8LvtHlOJpg106P0hX8a3NGhD0Oq9jGgmS77JwAOel36gncxou8CbEXlzpNsH65gJ2/tEJ3uSUZdHYQtQCiQtNzFbqI45JTu7UXkwgQv0Wxb7fnnNsz9StYmquVNp7MDw+Ol2/txA55tBBdSCeK+hy/X7cgI4tI/ftZ98c0YQbeUe/fdaV6PMxv0lPzED2We3N2Y080ineOib1ySsjY0diN2NM6NB8+9TJgqyLzd7X3WofJxExMFMRKuApCDCU7Bjw/FlKkISzvUOMmthL91LunQZ4x+ratiYVbz2cKa7PMcP0Qm7cFnosbyQKJWjM1KTOsCnoCS7iLZHxzKKE+prOWrssbcfENvpSTmouWrGppi9D8nwXDEFNyp+pVx8b7gwQ5GR/WjXb1PY/WkjBgNRaZbHzJlJzYTMBjNjlI1Nn0s4ncWhsZE=
```

Comment:
Hey @prograhammer  

How come you changed the key?

Reply:
Oh wow, no idea? I actually recently signed up with Travis with my own GitHub so maybe that's why? Let me put the original key back...